src/server/db/schema.ts

```diff
@@ -100,6 +100,7 @@ export const gender = pgEnum("gender", [
   "Male",
   "Non-Binary",
   "Other",
+  "Prefer not to answer (default)",
```

Comment:
under `levelOfStudy`, "Prefer not to answer" is used, should that be standard across all fields? rather than "Prefer not to answer (default)"?

Reply:
yeah ur right, good catch